libs/community/langchain_community/tools/databricks/tool.py

```diff
@@ -9,8 +9,9 @@
 from langchain_core.tools.base import BaseToolkit
 from typing_extensions import Self
 
+from databricks.sdk import WorkspaceClient
```

Comment:
This seems necessary to fix a pydantic forward reference issue; without it, calling `UCFunctionToolkit.include(...)` throws with


```
PydanticUserError: UCFunctionToolkit is not fully defined; you should define WorkspaceClient, then call UCFunctionToolkit.model_rebuild().
```